src/utils/dashlyApi.js

```diff
@@ -48,85 +48,48 @@ export const dashlyExchangeConnected = (exchangeType) => {
 
 /**
  * @param {UserEntity} userData User Entity.
+ * @param {'login'|'signup'} type Type of event to trigger .
  * @returns {void} None.
  */
-export const dashlyLogin = (userData) => {
+export const dashlyTrigger = (userData, type) => {
   const { dashly } = dashlyApi();
+  const eventToTrigger = type === "login" ? "$authorized" : "$registered";
 
   if (dashly) {
     // @ts-ignore
```

Comment:
We should try to avoid `@ts-ignore` as much as possible, especially now that we are open source. I know it takes more time and it's cumbersome with jsdoc but sometimes there are types packages already existing.